app/Http/Controllers/IncomingTextController.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Http\Requests\TwilioRequest as Request;
+
+class IncomingTextController extends Controller
+{
+    protected $commands = [
+        "unsubscribe" => [
+            "STOP",
+            "STOPAll",
+            "UNSUBSCRIBE",
+            "CANCEL",
+            "END",
+            "QUIT",
+        ],
+        "subscribe" => [
+            "START",
+            "YES",
+        ]
+    ];
+
+    /**
+     * Starting point for all incoming text messages.
+     */
+    public function receiveText(Request $request)
+    {
```

Comment:
If we also make commandForRequest default to 'unrecognized' then we can even get rid of the if and end up with
```php
return $this->commandForRequest($request)
```